README.md

```diff
@@ -29,14 +30,69 @@ Check the [wiki documentation][wiki] for more information on the modding API.
   [wiki]: https://github.com/ManuelBlanc/NecroLua/wiki
 
 ## Building from source
-To build NecroLua from source, run `nmake all`. You need the following software packages:
+1. First, install the following software packages:
 
 + Microsoft command line build tools (MSVC v142 - VS 2019 C++ x64/86 build tools)
-+ LuaJIT 2.1.0-beta3 (the libraries and the executable are both required)
-+ Microsoft Research Detours Package (https://github.com/microsoft/Detours)
+  + Make sure to use the x86 (32-bit) tools since NecroDancer.exe is a 32-bit executable
++ [LuaJIT 2.1.0-beta3][luajit]
+  + Use luajit's included `src/msvcbuild.bat` instead of running the makefile
++ [Microsoft Research Detours Package][detours]
+  + If you run into issues, try installing the ".NET desktop development tools" through the Visual Studio Installer
 + DbgHelp.dll – Windows Image Helper (included with Windows)
 + ImageMagick (_Optional_, used to make the app icon)
 
+  [luajit]: https://luajit.org/install.html
+  [detours]: https://github.com/microsoft/Detours
+
+2. After installing the above packages, go to your NecroLua directory and create an `include` subdirectory and a `lib` subdirectory, and copy in the required headers and library files; the following files should be present:
+
+```bash
+$ ls lib/ include/
+include/:
+detours.h  lauxlib.h  lua.h  luaconf.h  luajit.h  lualib.h
+
+lib/:
+detours.lib  lua51.dll  lua51.lib
```

Comment:
lua51.dll is not needed here, only with the NecroDancer.exe as you mention later.